app/lib/frontend/request_context.dart

```diff
@@ -46,6 +47,9 @@ class RequestContext {
   /// The status of the client session cookie.
   final ClientSessionCookieStatus clientSessionCookieStatus;
 
+  /// Whether to check rate limits for this request.
+  final bool checkRateLimits;
```

Comment:
How about making the rate-limit configurable?

So that it is specified in the config file... We have 3 limits... test probably only need to increase one or two of these limits.

Reply:
> Maybe we could add @visibleForTesting on the setter of the flag?

It's not only read in testing.